Makefile

```diff
@@ -5,7 +5,7 @@ CFLAGS-DEBUG=-O0 -Wall
 BIN=./bin
 
 SUDO=sudo
-MKLIB=$(SUDO) bash ./libs/install.sh ./libs
+MKLIB=bash ./libs/install.sh ./libs
```

Comment:
Totally agreed @ryukinix ! Thank you so much for your review! I will totally do it on #29! I've just reopened!